rpgJavaInterpreter-core/src/test/resources/smeup/MUDRNRAPU01101.rpgle

```diff
@@ -0,0 +1,8 @@
+     D £UIBDS          DS            10
+     D £UIBME                          0   INZ('test')
+     D XUIBME                          0   INZ('')
+
+     DXUIBME           S                   LIKE(£UIBME)
```

Comment:
You should add a comment that allows me to understand what you have fixed, for example the message error of jariko.